README.md

```diff
@@ -67,7 +67,7 @@ $ ./bin/generate-keys --tls-cert server.crt --tls-key server.key
 $ cp dendrite-config.yaml dendrite.yaml
 
 # Build and run the server:
-$ ./bin/dendrite-monolith-server --tls-cert server.crt --tls-key server.key --config dendrite.yaml
+$ go run ./cmd/dendrite-monolith-server --tls-cert server.crt --tls-key server.key --config dendrite.yaml
```

Comment:
`./build.sh` dumps binaries to `./bin` as per https://github.com/matrix-org/dendrite/blob/master/build.sh#L22 so this isn't really a fix for anything, just a different way of doing the same thing. I personally prefer the `go run` form though - thoughts @neilalexander ?

Reply:
This might be my lack of experience - I tried the go run command, and it didn't work, so asked in the dendrite-dev chatroom and was told to run the ./bin command instead